mail-client/claws-mail/claws_mail-4.1.0.recipe

```diff
@@ -0,0 +1,168 @@
+SUMMARY="A Lightweight and Fast GTK based Mail Client"
+DESCRIPTION="Claws Mail is a lightweight and highly \
+configurable email client and news reader \
+based on the GTK GUI toolkit, and runs on X, \
+Claws Mail capabilities are extended by plugins.\
+It comes with several plugins included, \
+all of which are built automatically \
+if the required libraries are present."
+HOMEPAGE=" https://www.claws-mail.org/"
+COPYRIGHT="1999-2022 the Claws Mail team and Hiroyuki Yamamoto"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://www.claws-mail.org/download.php?file=releases/claws-mail-$portVersion.tar.xz"
+CHECKSUM_SHA256="0e1a9ca0db8d2a9e058ae30cdc7fc919779214ec3c56ee0c8a7f88cc23817a8e"
+SOURCE_DIR="claws-mail-$portVersion"
+PATCHES="claws-mail-$portVersion.patchset"
+ADDITIONAL_FILES="claws-mail.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	claws_mail$secondaryArchSuffix = $portVersion
+	app:Claws_mail$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libarchive$secondaryArchSuffix
+	lib:libatk_1.0$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libdb$secondaryArchSuffix
```

Comment:
On 32bit I see libdb_6.0 not being found as a dependency for libetpan, so maybe add it here  and in BUILD_REQUIRES (works on 32bit):
```diff
-	lib:libdb$secondaryArchSuffix
+	lib:libdb_6.0$secondaryArchSuffix
```

Reply:
For this case, i thought it might be better to just upgrade from libdb-6.0 to libdb-6.1  and then rebuild libetpan against it, that will solve this issue without  tying it to any particular version, we already have the libdb-6.1 recipe, i tested it already on 32 bit it works with no changes, i don't know why -ldb is hard coded to -ldb6.0 in the libetpan-pc, maybe @korli  knows more about it, let's wait and see if it's okay to update to libdb-6.1.

Reply:
Well libetpan_devel should require devel:libdb then.

Reply:
You mean we have to use the versionned libdb in order to avoid adding devel:libdb as a dependency? i didn't know that, what should we do?

Reply:
Looking at the output from `pkg-config libetpan --libs,` I think the other devel packages should be declared there also?
For db you could add a versioning as depenency like: `devel:libdb$secondaryArchSuffix >= 6.0` (or 6.1)?
This should be done for libetpan.

Reply:
By other you mean these?
devel:libdb$secondaryArchSuffix >= 6.0 (or 6.1)
devel:libsasl2$secondaryArchSuffix
devel:libssl$secondaryArchSuffix

Reply:
Build laptop has been closed down earlier so can't check, but IIRC I also saw, libcurl, libcrypto and libiconv?

Reply:
FWIW, I tried a build with only adding `devel:libdb$secondaryArchSuffix >= 6.0 `in the recipe for libetpan (in `BUILD_REQUIRES` and `REQUIRES_devel`) and it worked ok without the change requested here.

Reply:
I've changed it to libdb-6.0, it's fine by me either way really, it's just that i don't understand why when libetpan is linked against libdb-6.0 we get -ldb-6.0 in the pkgconfig file, and when it is linked against libdb-6.1 we get just -libdb, so i thought why not just build claws-mail against the latest wihtout having to worry about any versionning.